Mathlib/Algebra/GroupWithZero/WithZero.lean

```diff
@@ -277,9 +274,7 @@ def withZeroUnitsEquiv {G : Type*} [GroupWithZero G]
   invFun a := if h : a = 0 then 0 else (Units.mk0 a h : Gˣ)
   left_inv := (by induction · <;> simp)
   right_inv _ := by simp only; split <;> simp_all
-  map_mul' x y := by
-    induction x <;> induction y <;>
-    simp [← WithZero.coe_mul, ← Units.val_mul]
+  map_mul' := (by induction · <;> induction · <;> simp [← WithZero.coe_mul])
```

Comment:
Do you need the outer parentheses?

Reply:
Yes, to make the `·` legal. I think this type of code is probably bad style though, but since this matches the `left_inv` field, cleaning it up seems out of scope.